Components/Badge/badge.css

```diff
@@ -41,19 +40,19 @@
     padding:7px;
     border:2px solid white
 }
-.avatar {
+.avatar-badge {
     width: 50px;
     height: 50px;
     object-fit: fill;
     border-radius: 50%;
 }
 
 .online{   
-    background-color:#22C55E;    
+    background-color:var(--success);    
 }
 .offline{   
-    background-color:grey;   
+    background-color:var(--basic);   
 }
 .dnd{  
```

Comment:
Write a specific class name. BCS it's a component library. '.dnd' doesn't make sense.

Reply:
okay, will do that!